CHANGES/7691.bugfix

```diff
@@ -0,0 +1 @@
+Fix issue when .treeinfo file is missing but http returns 'access denied' instead 'file not found' (http server autoindex disabled).
```

Comment:
Should we swap the commit message and the changelog entry? :D

"Added handling of HTTP 403 Forbidden during DistributionTree detection." as a changelog message for users.

"When .treeinfo file is missing, server returns HTTP 403 instead of HTTP 404 if the autoindex is disabled on the server)." as a commit message. 

Thoughts?